zerver/migrations/0579_userprofile_allow_private_data_export_and_more.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 5.0.8 on 2024-09-09 08:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0578_namedusergroup_deactivated"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="realmuserdefault",
+            name="allow_private_data_export",
+            field=models.BooleanField(default=False),
```

Comment:
We don't show this in `organization > default user settings`. This is present in `realmuserdefault` table to keep our code aligned with the `property_types` code flow. We do similar thing for `enable_login_emails` setting.

Reply:
Yep, sounds correct.